chat/src/main/java/com/twilio/video/app/chat/ChatManagerImpl.kt

```diff
@@ -56,6 +60,16 @@ class ChatManagerImpl(
         conversationsClientWrapper.create(context, token, props, conversationsClientCallback)
     }
 
+    override fun sendMessage(message: String) {
+        if (chatState.value.connectionState == Connected) {
+            val options = messageWrapper.options()
+                    .withBody(message)
+            conversation?.sendMessage(options, sendMessageCallback)
+        } else {
+            throw IllegalStateException("Cannot send a message while not in the connected state")
```

Comment:
Adding the current state will help debug.

Reply:
Good catch thanks!